src/views/tenants/tenants.jsx

```diff
@@ -49,6 +52,33 @@ const getDisplayTenants = (tenants, showHoused, showArchived) => {
   return tenants.filter(tenant => (showHoused || tenant.lease) && (showArchived || !tenant.archived));
 };
 
+const expandRow = isSmallScreen => ({
```

Comment:
Same here

Reply:
Unfortunately, we can't, as it uses a hook as a parameter, and hooks have to be called from their components.